# Save the map to the filesystem
std_msgs/String filename 